include/lemlib/devices/encoder/motor.hpp

```diff
@@ -15,14 +16,14 @@ class MotorEncoder : public Encoder {
          * @param motors pointer to the motor group to be used
          * @param rpm output rpm
          */
-        MotorEncoder(std::shared_ptr<pros::MotorGroup> motors, float rpm);
+        MotorEncoder(std::shared_ptr<pros::MotorGroup> motors, AngularVelocity rpm);
```

Comment:
```suggestion
        MotorEncoder(std::shared_ptr<pros::MotorGroup> motors, AngularVelocity speed);
```
Update parameter name for consistency